### PR TITLE
Cs/91144282 remove ag deps

### PR DESCRIPTION
**Pull request by wolves**
https://www.pivotaltracker.com/story/show/91144282

### CODE REVIEW

File: app/coffeescript/components/utils/cluster_opts.coffee [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+#
+# BEWARE: This is not full proof since agent strings can be masqueraded.
+#  Depending on your need this may or may not work
+#
```
Comment (warrenv) [inline app/coffeescript/components/utils/cluster_opts.coffee]: Are these comments relevant Mr copy-and-paste?

File: app/coffeescript/components/utils/cluster_opts.coffee [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+define [ 'flight/lib/component' ], ( defineComponent ) ->
```
Comment (warrenv) [inline app/coffeescript/components/utils/cluster_opts.coffee]: this is not a flight component so you don't have to pull it in here.

File: app/coffeescript/components/ui/clusters.coffee [hunk 3]
```diff
@@ -5,25 +5,23 @@ define [
...
   compose,
   defineComponent,
   markerClusterer,
   map_utils,
-  mobileDetection
+  mobileDetection,
+  clusterOpts
 ) ->
 
   initMarkerClusters = ->
```
Comment (lfender6445) [inline app/coffeescript/components/ui/clusters.coffee:17]: i might be a little foggy on the stories AC - is goal to be able to pass in custom options to market cluster?

Comment (wolves) [inline app/coffeescript/components/ui/clusters.coffee:17]: Originally the story was to remove ag deps from the file, but being that there wasn't much to be removed in this particular file @mmaiza and I discussed cleaning up some of the opts being set in the file into a mixin. I did it as a component but will be refactoring to a mixin before updating the PR

**Pull request closed without merging**